content/content-strategy/manage-content-requests/create-content/why.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: text/textblock
+---
```

Comment:
missing `section` param and value: needs to relate to the same param in `content/content-strategy/manage-content-requests/create-content/why-head.md`